### PR TITLE
Add VSCode Jest Debug script

### PR DESCRIPTION
**Pull request by richardtagger**

### CODE REVIEW

**Comment by richardtagger** [comment]
@unindented

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 /docs
 /node_modules
 /config/travis/deploy.key
-.*
 *.diff.png
 *.new.png
+.DS_Store
+/.jest
```
Comment (unindented) [inline .gitignore]: Why not ignore all files and folders starting with dot?

Comment (richardtagger) [reply .gitignore]: Cos we want the .vscode/ folder? Looking at YamJS' we manually list them

Comment (unindented) [reply .gitignore]: `git add .vscode -f`, and keep the `.gitignore` as it was.

Comment (unindented) [reply .gitignore]: Things like `.DS_Store` shouldn't live in a project's `.gitignore`. They should live in each person's global `.gitignore`. YamJS is not a good guide for anything.

**Comment by richardtagger** [comment]
ok done

**Pull request merged**